org.eclipse.scanning.event/src/org/eclipse/scanning/event/ConsumerImpl.java

```diff
@@ -643,7 +616,7 @@ public void resume() throws EventException {
 			awaitPaused = false;
 			// We don't have to actually start anything again because the getMessage(...) call reconnects automatically.
 			paused.signalAll();
-			logger.info(getName()+" running");
+			LOGGER.info(getName()+" running");
 			System.out.println(getName()+" running");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)